kernel/classes/datatypes/ezimage/ezimagetype.php

```diff
@@ -21,6 +21,8 @@ class eZImageType extends eZDataType
 {
     const FILESIZE_FIELD = 'data_int1';
     const FILESIZE_VARIABLE = '_ezimage_max_filesize_';
+    const ALTTEXTREQUIRED_FIELD = 'data_int2';
+    const ALTTEXTREQUIRED_VARIABLE = 'ContentAttribute_alttextrequired_checked';
```

Comment:
Is this supposed to include "ContentAttribute"? Seems like the convention is to omit this and then prepend a $base string